packages/bcer-api/app/src/location/location.service.ts

```diff
@@ -88,7 +88,16 @@ export class LocationService {
     qb.where('location.noi IS NOT NULL');
 
     if (query.search) {
-      qb.andWhere('(LOWER(location.addressLine1) LIKE :search OR LOWER(location.doingBusinessAs) LIKE :search OR LOWER(business.legalName) LIKE :search OR LOWER(business.businessName) LIKE :search)', { search: `%${query.search.toLowerCase()}%` });
+      qb.andWhere(`
+        (
+          LOWER(location.addressLine1) LIKE :search OR
+          LOWER(location.doingBusinessAs) LIKE :search OR
+          LOWER(location.city) LIKE :search OR
+          REPLACE(LOWER(location.postal), ' ', '') LIKE REPLACE(:search, ' ', '') OR
```

Comment:
Just added this ' ' -> '' transformation to allow postal code searches for both `V8V4Z3` and `V8V 4Z3`